packages/editor/src/components/blocks/datepicker/DatePicker.tsx

```diff
@@ -0,0 +1,49 @@
+import type { DatePicker, Prettify } from '@axonivy/form-editor-protocol';
+import type { ComponentConfig, UiComponentProps } from '../../../types/config';
+import './DatePicker.css';
+import { baseComponentFields, defaultBaseComponent } from '../base';
+import IconSvg from './DatePicker.svg?react';
+
+type DatePickerProps = Prettify<DatePicker>;
+
+export const defaultInputProps: DatePicker = {
+  label: 'Label',
+  value: '',
+  datePattern: 'dd.MM.yyyy',
+  timePattern: 'HH:mm',
+  showTime: false,
+  ...defaultBaseComponent
+} as const;
+
+export const DatePickerComponent: ComponentConfig<DatePickerProps> = {
+  name: 'DatePicker',
+  category: 'Elements',
+  subcategory: 'Interactions',
+  icon: <IconSvg />,
+  description: 'A simple datepicker for Date or DateTime',
+  defaultProps: defaultInputProps,
+  render: props => <UiBlock {...props} />,
+  create: ({ label, value, ...defaultProps }) => ({ ...defaultInputProps, label, value, ...defaultProps }),
+  outlineInfo: component => component.label,
+  fields: {
+    label: { subsection: 'General', label: 'Label', type: 'text' },
+    value: { subsection: 'General', label: 'Value', type: 'textBrowser', options: { onlyTypesOf: 'Date' } },
+    datePattern: { subsection: 'General', label: 'Date Pattern', type: 'text', options: { placeholder: 'e.g. dd.MM.yyyy' } },
+    showTime: { subsection: 'General', label: 'Show Time', type: 'checkbox' },
+    timePattern: {
+      subsection: 'General',
+      label: 'Time Pattern',
+      type: 'text',
+      options: { placeholder: 'e.g. HH:mm' },
+      hide: data => !data.showTime
+    },
+    ...baseComponentFields
+  }
+};
+
+const UiBlock = ({ label, value, datePattern, timePattern, showTime }: UiComponentProps<DatePickerProps>) => (
+  <div className='block-input'>
+    <span className='block-input__label'>{label}</span>
+    <span className='block-input__input'>{value === '' ? (showTime ? datePattern + ' ' + timePattern : datePattern) : value}</span>
+  </div>
```

Comment:
I would display a calendar icon or something here too, so it is easier to recognize that this is a datepicker

Reply:
thanks, i have added the calender Icon like this 👍:
![grafik](https://github.com/user-attachments/assets/0e064158-0d60-4f40-b963-055009727b54)